content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'How to Add Dependencies or Outputs to a Stage'
-description: 'It's possible to add files or directories to existing stages, with
-or without executing them.'
+description: "It's possible to add files or directories to existing stages,
+with or without executing them."
```

Comment:
p.s. @rogermparent any way you can think that we could catch these in the PR checks? It wasn't in https://github.com/iterative/dvc.org/pull/1976/checks?check_run_id=1465824496 (right before merge). Thanks!

Reply:
This is certainly something I should look into! To be specific, are we talking about checking for lint/format issues on MD frontmatter?

Reply:
@rogermparent the issue above was that I put a single-quote inside the single-quoted string. Seemed to work locally so I missed it, and it passed the PR checks, but it failed to build of course.